Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/Sync/ChangeEventUnitTests.swift

```diff
@@ -107,6 +121,61 @@ class ChangeEventUnitTests: XCTestCase {
                                                                                      documentId: docId,
                                                                                      writePending: writePending)
 
-        XCTAssertEqual(expectedLocalDeleteEvent, actualLocalDeleteEvent)
+        try compare(expectedChangeEvent: expectedLocalDeleteEvent,
+                    to: actualLocalDeleteEvent)
+    }
+
+    func testRoundTrip() throws {
```

Comment:
[minor] can you add code to test the round trip for a change event with an unknown change event? It can be a `"blah"` event going in, and should be an `"unknown"` event coming out.

Reply:
I already did.

Reply:
🤦‍♂️  my bad